samples/ClassroomLabs/Scripts/HyperV/SetupForNestedVirtualization.ps1

```diff
@@ -283,7 +320,14 @@ try {
         Write-Host "Unable to set WinNat service to Automatic.  Hyper-V virtual machines will not have internet connectivity when service is not running." -ForegroundColor Yellow
     }  
 
-    # Tell the user script is done.
+    # Tell the user that VMs will need to set their own IPs on Windows 10
+    # Link for Windows machines https://support.microsoft.com/en-us/help/15089/windows-change-tcp-ip-settings
+    if ($(Get-RunningServerOperatingSystem ) -eq $false) {
+        Write-Host -Object "DHCP Server is not supported on Windows 10. `
+        IP configuration will need to be done manually from within the VM itself `

```

Comment:
What do you think about changing the warning to something like
"IP addresses for each Hyper-V VM.  Address must be 192.168.0.4 and 192.168.0.254 with a subnet mask of 255.255.255.0.  To use the Azure Dns use 168.63.129.16"

This would avoid people have to grep through the script to find the values.

Reply:
Agreed, I've made the change.